ocs_ci/utility/azure_utils.py

```diff
@@ -39,7 +40,20 @@ def load_cluster_resource_group(cluster_path, terraform_filename=TERRRAFORM_FILE
     Returns:
         string with resource group name
     """
-    filepath = os.path.join(cluster_path, terraform_filename)
+    terraform_files = [
+        os.path.join(cluster_path, f)
+        for f in [OLD_TERRRAFORM_FILENAME, TERRRAFORM_FILENAME]
+    ]
+    filepath = None
+    for tf_file in terraform_files:
+        if os.path.exists(tf_file):
+            filepath = os.path.join(cluster_path, tf_file)
```

Comment:
I think we can avoid cluster_path as argument of function and read it directly from config.